xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+  <para>
+   &deng; provides you with the following advantages:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Isolation of applications and operating systems through containers.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Near native performance, as &deng; manages allocation of resources in real
+     time.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Controls network interfaces and resources available inside containers
+     through cgroups.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Versioning of images.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Allows building new images based on existing ones.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Provides you with container orchestration.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.docker.concept.limitations_deng">
+  <title>Limitations of &deng;</title>
+  <para>
+   On the other hand, &deng; has the following limitations:
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+   <listitem>
+    <para>
+     Containers run inside the host system's kernel and cannot use a different
+     kernel.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Only allows Linux <emphasis>guest</emphasis> operating systems.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     &deng; is not a full virtualization stack like &xen;, &kvm;, or &libvirt;.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Security depends on the host system. Refer to the
+     <link xlink:href="http://docs.docker.com/articles/security/">official
+      security documentation</link> for more details.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.docker.concept.images-applications">
+  <title>&docker; Images of Applications</title>
+  <para>
+  &deng; is a technology that can help minimize resources used to run or
+  build applications. There are several types of applications that are
+  suitable to run inside a &docker; container like daemons, Web pages or
+  applications that expose ports for communication. You can use &deng; to
+  automate building and deployment processes by adding the build process
+  into a &docker; image, then building the image and then running containers
+  based on that image.
+ </para>
+ <sect2 xml:id="sec.docker.concept.images-applications.run-inside">
+  <title>Advantages of Running Applications Inside &docker; Containers</title>
+  <para>
+   Running an application inside a &docker; container has the
+   following advantages:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     You can minimize the runtime environment of the application as you can add
+     to the &docker; image of the application just the required processes and
+     applications.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The image with your application is portable across machines also with
```

Comment:
"is portable" -> phrase verbally: "You can port the image with your application across machines with different Linux systems."

Reply:
This isn't porting...it is consumable across container engines so it literally is "portable" not ported.